test/built-ins/Object/prototype/toString/symbol-tag-non-str-proxy-function.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2016 the Apple Inc. All rights reserved.
```

Comment:
Is this correct?

Reply:
Pretty much, yeah: this test case (deleted `@@toStringTag`) was originally submitted in #755, I've just moved it to separate file and added async function case.